pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -187,6 +185,7 @@ def on_validation_end(self, trainer, pl_module):
 
     def on_save_checkpoint(self, trainer, pl_module) -> Dict[str, Any]:
         return {
+            "best_model_monitor": self.monitor,
```

Comment:
this can just be `monitor` right? not sure what makes it the "best" since it'll be the same value in each checkpoint
```suggestion
            "monitor": self.monitor,
```

Reply:
It can, kept the prefix for consistency. The best refers to "the monitor of the best model" not "the best monitor of the model".

Im fine with changing it anyways.

Reply:
just `monitor` is fine I think or maybe `checkpoint_monitor`.

Reply:
@carmocca are you saying the monitor could change from iteration to iteration? my first choice would also have been just monitor.